### ISSUE TITLE
tsdump is missing percentile metrics

### ISSUE THREAD

**Issue by sravotto** [cockroachdb/cockroach#106448]
**Describe the problem**

tsdump is missing the following percentile metrics:

```
admission.elastic_cpu.over_limit_durations
 admission.wait_durations.elastic-cpu
 admission.wait_durations.elastic-cpu.bulk-normal-pri
 admission.wait_durations.elastic-cpu.normal-pri
 admission.wait_durations.kv-stores.bulk-normal-pri
 admission.wait_durations.kv-stores.high-pri
 admission.wait_durations.kv-stores.locking-pri
 admission.wait_durations.kv-stores.normal-pri
 admission.wait_durations.kv-stores.ttl-low-pri
 admission.wait_durations.kv.bulk-normal-pri
 admission.wait_durations.kv.high-pri
 admission.wait_durations.kv.locking-pri
 admission.wait_durations.kv.normal-pri
 admission.wait_durations.sql-kv-response.locking-pri
 admission.wait_durations.sql-kv-response.normal-pri
 admission.wait_durations.sql-leaf-start.locking-pri
 admission.wait_durations.sql-leaf-start.normal-pri
 admission.wait_durations.sql-root-start.locking-pri
 admission.wait_durations.sql-root-start.normal-pri
 admission.wait_durations.sql-sql-response.locking-pri
 admission.wait_durations.sql-sql-response.normal-pri
 changefeed.nprocs_consume_event_nanos
 changefeed.nprocs_flush_nanos
 go.scheduler_latency
 jobs.row_level_ttl.delete_duration
 jobs.row_level_ttl.select_duration
 jobs.row_level_ttl.span_total_duration
 rebalancing.replicas.cpunanospersecond
 rebalancing.replicas.queriespersecond
 sql.mem.internal.session.prepared.max
 sql.mem.sql.session.prepared.max
 sql.pre_serve.mem.max
 storage.wal.fsync.latency
```
**To Reproduce**

Obtain a tsdump and collect the p50 percentiles 

```
cockroach debug tsdump --insecure --from "...."  | grep p50
```

**Expected behavior**
The above metrics should be present in the tsdump

**Environment:**
 - CockroachDB version v21.3.2

Jira issue: CRDB-29567

**Comment by ericharmeling** [comment]
@sravotto Can you confirm the version of crdb for this issue?

**Comment by sravotto** [comment]
Fixed the version (sorry for the typo); also on V23.1.5
<img width="1662" alt="image" src="https://github.com/cockroachdb/cockroach/assets/8729415/c3354dec-e9c2-4111-a483-f6a9de5777f2">

```
cockroach debug tsdump --host... --insecure  | grep go.scheduler_latency | wc -l
0
```

**Comment by ericharmeling** [comment]
@sravotto this should be fixed in 23.1.10
